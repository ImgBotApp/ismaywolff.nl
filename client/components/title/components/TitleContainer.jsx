import glamorous from 'glamorous'

const TitleContainer = glamorous.div({
  margin: '2rem 0'
})

export default TitleContainer

