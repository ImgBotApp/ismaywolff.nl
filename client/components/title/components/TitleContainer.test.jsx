import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import TitleContainer from './TitleContainer'

describe('<TitleContainer />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<TitleContainer />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
