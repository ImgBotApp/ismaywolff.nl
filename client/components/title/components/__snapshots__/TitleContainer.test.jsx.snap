// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TitleContainer /> renders correctly 1`] = `
<div
  className="fLnhrp"
/>
`;
